stock_by_warehouse/static/src/xml/template.xml

```diff
@@ -30,4 +30,37 @@
             </div>
         </div>
     </t>
+
+    <t t-name="StockAvailabilityAlert">
+        <p class="availability_product my-0 mr-2">
+            The product is available in: <t t-raw="locations_available"/> locations.
+            <a role="button" tabindex="0" class="js_available_info fa fa-bars"></a>
```

Comment:
What do you think about using `href="#"` instead of `tabindex="0"`?

Reply:
tabindex is to be the first pre-selected elemente, no relation with href @luisg123v